src/web/components/BusinessContacts/BusinessContactDialog.tsx

```diff
@@ -47,7 +47,7 @@ function BusinessContactDialog({
             rules={{
               required: 'Please specify email alias.',
               pattern: {
-                value: /\S+@\S+\.\S+/,
+                value: /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/,
```

Comment:
Can we extract this to a variable? I'm sure I'm not alone in struggling to understand regex :D

Reply:
Variable extracted and reused